src/Build/Evaluation/ProjectRootElementCache.cs

```diff
@@ -123,24 +122,6 @@ internal class ProjectRootElementCache : ProjectRootElementCacheBase
         /// </summary>
         private Object _locker = new Object();
```

Comment:
I think this is a singleton and therefore will need to have a static constructor (even if it is empty). see https://csharpindepth.com/articles/Singleton